java/clients/src/test/resources/reports/ingest/job/json/bulkImport/acceptedJob.json

```diff
@@ -0,0 +1,20 @@
+{
+  "queueMessages": {
+    "ingestMessages": 0
+  },
+  "jobList": [
+    {
+      "jobId": "job11111-1111-1111-1111-111111111111",
+      "jobRunList": [
+        {
+          "validatedStatus": {
```

Comment:
It would be nice to ensure that we always show all the updates in a run, even if there is somehow more than one of the same type.